modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx

```diff
@@ -50,104 +51,129 @@ const MDFClaimPage = ({
 		)
 	);
 
-	const checkClaimLimit =
-		mdfRequest.mdfRequestToMdfClaims &&
-		mdfRequest?.mdfRequestToMdfClaims?.length >= 2
-			? true
-			: false;
-
-	return (
-		<PRMForm name="New" title="Reimbursement Claim">
-			<PRMForm.Section
-				subtitle="Check each expense you would like claim and please provide proof of performance for each of the selected expenses."
-				title={`${mdfRequest?.overallCampaignDescription} (${mdfRequest?.id})`}
-			>
-				<p className="font-weight-bold my-4 text-paragraph">
-					Upload Proof of Performance Documents
-					<span className="text-danger">*</span>
-				</p>
-
-				{values.activities?.map((activity, index) => (
-					<ActivityClaimPanel
-						activity={activity}
-						activityIndex={index}
-						key={`${activity.id}-${index}`}
-						overallCampaignDescription={
-							mdfRequest.overallCampaignDescription
+	const claimsNotDraft = mdfRequest.mdfRequestToMdfClaims?.filter(
+		(mdfRequestToMdfClaim) => {
+			return !mdfRequestToMdfClaim.claimStatus.includes(Status.DRAFT);
+		}
+	).length;
+
+	const getClaimPage = (claimsNotDraft?: number) => {
```

Comment:
Since this variable can be accessed globally, I think there is no need to be passing it to the function
```suggestion
	const getClaimPage = () => {
```